src/modules/main-nav/MainNav.scss

```diff
@@ -1,4 +1,200 @@
-.main-nav {
-  background-color:yellow;
-  color: blue;
+ @import './../../shared/utils';
+ @import './../../shared/variables';
+
+.main-nav{
+	width: 100vw;
+	height: 100vh;
+}
+
+.main-nav-container{
+	width: 100%;
+	height: 20%;
+	background-color: white;
+	display: flex;
+	align-content: center;
+	justify-content: center;
+
+	&__button{
+		display: flex;
+		align-items: center;
+		width: 20%;
+		height: 100%;
+		padding: 0 20px;
+		position: relative;
+
+		@include large-screen{
+			display: none;
+		}
+	}
+
+	.button-closed {
+		display: block;
+		width: 40px;
+		height: 40px;
+		border-radius: 2px;
+		background: darken(white, 5%);
+		border: 1px solid darken(green, 15%);
+
+		&-line {
+			display: block;
+			height: 2px;
+			width: 100%;
+			background: black;		
+		
+			&::before,
+			&::after {
+				display: block;
+				content: '';
+				height: 2px;
+				width: 100%;
+				background: black;
+				position: relative;
+			}
+		
+			&::before {
+				top: -6px;
+			}
+		
+			&::after {
+				bottom: -4px;
+			}
+		}
+
+		&-cross{
+			position: relative;
+  			display: inline-block;
+  			width: 30px;
+  			height: 30px;
+  			overflow: hidden;
+
+  			&::before, &::after {
+    			content: '';
+    			position: absolute;
+    			height: 2px;
+    			width: 100%;
+    			top: 50%;
+    			left: 0;
+    			margin-top: -2px;
+    			background: #000;
+  			}
+
+  			&::before {
+    			transform:rotate(45deg);
+  			}
+  			
+  			&::after {
+    			transform:rotate(-45deg);
+  			}
+		}	
+	}
+
+	.main-nav-container__logo{
+		width: 80%;
+		height: 100%;
+
+		.logo{
+			width: 20vh;
+			height: 90%;
+		}
+	}
+
+}
+
+.container{
```

Comment:
Having in mind that this files related to the MainNav, ideally everything in here should be related to that. `.container` seems to be too generic.